sandbox_dns_unix.go

```diff
@@ -0,0 +1,323 @@
+// +build !windows
```

Comment:
Let's not tag them as `!windows` because that is not true. Instead tag them as `linux, freebsd`